css/commentstyle.css

```diff
@@ -0,0 +1,30 @@
+.comment-div {
```

Comment:
[OPTIONAL] When you open the modals with comments, these open always at the top of the page. So if you have scrolled down, you won't see it until you scroll up again. I think you should `fix` the modal to the view so no matter where the user is it opens in front of they.